pkg_auto/impl/pkg_auto_lib.sh

```diff
@@ -2255,26 +2160,37 @@ function handle_pkg_update() {
     local pkg_name
     pkg_name=${new_pkg#*/}
     local -a lines
-    lines=( "from ${old} to ${new}")
+    lines=( "0:from ${old} to ${new}")
```

Comment:
It's not clear to me why `0:` was added to all these lines. Would some example output make it clearer?

Reply:
It's a number:text pair - the number is the indentation level. It's done this way, because they will also contain lines from `diff_report` (top of the `md5_cache_diff_lib.sh`) which stores its lines in that way. The `generate_summary_stub` function handles those lines. So the example output of `lines=( "0:first" "1:second" )` could be:

```
- first
  - second
```